app/src/main/java/com/fastaccess/ui/modules/search/SearchUserActivity.kt

```diff
@@ -0,0 +1,95 @@
+package com.fastaccess.ui.modules.search
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.text.Editable
+import butterknife.ButterKnife
+import butterknife.OnClick
+import butterknife.OnEditorAction
+import butterknife.OnTextChanged
+import com.fastaccess.R
+import com.fastaccess.helper.AnimHelper
+import com.fastaccess.helper.InputHelper
+import com.fastaccess.helper.Logger
+import com.fastaccess.ui.modules.search.repos.SearchReposFragment
+import kotlinx.android.synthetic.main.activity_search_user.*
+
+class SearchUserActivity : AppCompatActivity() {
```

Comment:
why not extending BaseActivity? Every activity in the app should be extending BaseActivity where it handles themes & alot more things!.

Reply:
Extending baseActivity requires making new Presenter and View Classes, which I figured weren't really needed for such a small activity. Will change accordingly.
Should I just add Null to the presenter/View or make them according to the activity at hand?

Reply:
then simply use `SearchUserActivity<BaseMvp.View, BasePresenter<BaseMvp.View>` and in provide Presenter = `BasePresenter<BaseMvp.View>()`